cypress-tests/readme.md

```diff
@@ -72,6 +72,32 @@ To run test cases, follow these steps:
    npm run cypress:routing
    ```
 
+In order to run cypress tests against multiple connectors at a time:
+
+1. Set up `.env` file that exports necessary info:
+
+   ```env
+   export DEBUG=cypress:cli
+
+   export CYPRESS_ADMINAPIKEY='admin_api_key'
+   export CYPRESS_BASEURL='base_url'
+   export CYPRESS_CONNECTOR_AUTH_FILE_PATH="path/to/creds.json"
+
+   export PAYMENTS="payment_connector_1 payment_connector_2 payment_connector_3 payment_connector_4"
```

Comment:
Can you update the docs with the updated variable names, `PAYMENTS_CONNECTORS` and `PAYOUTS_CONNECTORS`?